### ISSUE TITLE
Add Range (1-10 etc) Support

### ISSUE THREAD

**Issue by XxUnkn0wnxX** [bucanero/apollo-lib#19]
Hey @bucanero would it be possible to add the option to allow for selecting a set amount of cheats in one go rather than typing every value like this `1,2,3,4,5,6,7,8,9,10`

like say, if I wanted all cheats from 5 to 20, I could do `5-20` and all cheats from 5 to 20 would get applied.
since I tend to make long cheat lists or groups with like reset all starts for the 1st 10, next 10 some other values, next 10 other values etc..

**Comment by bucanero** [comment]
It could be added, but I'm working on many other things across other platforms, and Apollo CLI is probably a very small tool that only a few advanced users can actually understand and use properly. So I can't really say when/if I'll have to do take a look at this. If someone wants to submit a PR I'll be happy to review and merge it. 

(to be honest, I think that getting a GUI is probably better in the long run, and I saw someone working on a GUI add-on)

**Comment by XxUnkn0wnxX** [comment]
yep works fine, but CLI is still way faster for rapid mass testing codes with.
GUI hope it's cross-platform.. I'm not much of a windows fan..
